blitz/array/fastiter.h

```diff
@@ -286,7 +286,7 @@ class FastArrayIteratorBase {
             str += ",";
 
             char tmpBuf[10];
-            sprintf(tmpBuf, "%d", N_rank);
+            snprintf(tmpBuf, 10, "%d", N_rank);
```

Comment:
Thanks! Let's perhaps create a preprocessor constant with this value of 10 and use it in all 7 places mentioned in this PR - to make it more readable and less bug/typo prone.